lib/spider/spider/in_context_learning.py

```diff
@@ -0,0 +1,128 @@
+import json
+import os
+
+import clearml
+import hydra
+import nltk
+import torch
+import transformers
+import wandb
+from clearml import Task
+from omegaconf import OmegaConf
+from tqdm import tqdm
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from spider._path import _ROOT, DATASET_PATH
+from spider.clearml_utils import setup_clearml
+from spider.evaluation import build_foreign_key_map_from_json, evaluate
+from spider.utils import extract_sql, is_valid_sql
+
+nltk.download("punkt")
+
+
+OUTPUT_FILE = DATASET_PATH / "output.txt"
+
+gold = DATASET_PATH / "dev_gold.sql"
+db_dir = DATASET_PATH / "database"
+table = DATASET_PATH / "tables.json"
+
+
+@hydra.main(version_base="1.3", config_path="../configs", config_name="eval.yaml")
```

Comment:
The config path should be absolute, so that it doesn't get broken if we move the script! You can use `_ROOT / "configs"` for instance

Reply:
yes